.github/workflows/macos_dependencies.txt

```diff
@@ -0,0 +1,167 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: osx-64
+@EXPLICIT
+https://conda.anaconda.org/conda-forge/osx-64/pandoc-2.19.2-h694c41f_1.tar.bz2
```

Comment:
I'm not familiar with this type of file, so I'm wondering why this file and not an environment.yml file?